app/lib/efile/nj/nj_property_tax_eligibility.rb

```diff
@@ -0,0 +1,38 @@
+module Efile
+  module Nj
+    module NjPropertyTaxEligibility
+      INELIGIBLE = :ineligible
+      INELIGIBLE_FOR_DEDUCTION = :ineligible_for_deduction
+      NOT_INELIGIBLE = :not_ineligible
+
+      class << self
+        def determine_eligibility(intake)
+          state_wages = Efile::Nj::NjStateWages.calculate_state_wages(intake)
+
+          wage_minimum = if intake.filing_status == :married_filing_separately || intake.filing_status == :single
+            10_000
+          else
+            20_000
+          end
+
+          return NOT_INELIGIBLE if state_wages > wage_minimum
```

Comment:
[dust] The double negative is throwing me. I wonder if an alternative like "possibly eligible," or "eligible for credit and deduction" or something else more specific might make this easier to reason through

Reply:
yeah i considered "possibly eligible" -- since actual eligibility depends on additional answers to questions we can't just simply say "eligible" for anything yet. agree that double-negative is not great. i think i'll change to "possibly eligible"

Reply:
[DUST] Agree with Melanie here, possibly eligible feels better to me.

Reply:
changed to

```
INELIGIBLE = :ineligible
POSSIBLY_ELIGIBLE_FOR_CREDIT = :possibly_eligible_for_credit
POSSIBLY_ELIGIBLE_FOR_DEDUCTION_OR_CREDIT = :possibly_eligible_for_deduction_or_credit
```